common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4632,4 +4634,12 @@ object CelebornConf extends Logging {
       .version("0.5.0")
       .intConf
       .createWithDefault(8)
+
+  val WORKER_APPLICATION_REGISTRY_CACHE_SIZE: ConfigEntry[Int] =
+    buildConf("celeborn.worker.applicationRegistryCache.size")
+      .categories("worker", "auth")
+      .doc("Cache size of the application registry on Workers.")
+      .version("0.5.0")
+      .intConf
+      .createWithDefault(10000)
```

Comment:
Why does the default value of cache size is 10000?

Reply:
I am open to a better default. We have more than 200K apps running per day. The number of apps active at a given will be much smaller than that. I thought 10K is a conservative default and approx will take a few MBs.

Reply:
workerInfo.getApplicationIdSet changes with new apps being added or removed. I am not sure how we could configure the default value of this config to be the size of that set. I did update the code to unregister an app from the worker's secretRegistry from the cleaner thread.